explorer/websocket.go

```diff
@@ -325,10 +325,10 @@ func (wsh *WebsocketHub) run() {
 	} // for {
 }
 
-// MaybeSendTxns adds a mempool transaction to the client broadcast buffer. If
+// maybeSendTxns adds a mempool transaction to the client broadcast buffer. If
```

Comment:
This has no use being exported since it is only called from `(*WebsocketHub).run`.